src/schemas/21-686C/schema.js

```diff
@@ -98,6 +98,10 @@ let schema = {
   additionalProperties: false,
   definitions: _.merge(definitions,
     {
+      phone: {
+        type: 'string',
+        pattern: '^[-0-9]{10,12}$'
```

Comment:
do we not have a more robust regex for phone numbers? seems like this would allow a lot of invalid phone numbers through.

Reply:
Yeah, this is a pretty lazy regex.  I'm going to use our predefined `usaPhone` definition (i.e. allows 10 digits) and strip out dashes when prefilling.